pages/singlesite/[slug].js

```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[2]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[2]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[3]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[3]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <StyledFigure>
+                <StyledImage
+                  className="ancient-picture"
+                  src={currentSite.ancientPicture[4]}
+                  alt={currentSite.adress}
+                  height={200}
+                  width={300}
+                />
+                <figcaption className="figcaption-ancient">
+                  {currentSite.ancientCaption[4]}
+                </figcaption>
+              </StyledFigure>
+            </StyledSwiperSlide>
+            <StyledSwiperSlide>
+              <div className="credits-card">
+                <h2>
+                  {currentSite.creditsCaption + ": "}
+                  {currentSite.coordinates[0] +
+                    " | " +
+                    currentSite.coordinates[1]}
+                </h2>
+                <p className="credits">
+                  {currentSite.credits[0]}
+                  <br />
+                  {currentSite.credits[1]}
+                  <br />
+                  {currentSite.credits[2]}
+                  <br />
+                  {currentSite.credits[3]}
+                  <br />
+                  {currentSite.credits[4]}
+                </p>
+              </div>
+            </StyledSwiperSlide>
+          </Swiper>
+        </StyledSwiperSlide>
+      </Swiper>
+
+      <StyledLink href="/singlesite">
+        <Image
+          src={"/profan--rosybrown.svg"}
+          alt="Profan-Logo"
+          height={50}
+          width={50}
+        />
+      </StyledLink>
+    </StyledContainer>
   );
 }
 
+const StyledSwiperSlide = styled(SwiperSlide)`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+`;
+
 const StyledImage = styled(Image)`
-  object-fit: cover;
-  z-index: -1;
+  display: block;
+  object-fit: contain;
+  width: 80% !important;
+  height: 80% !important;
```

Comment:
I know it works. 
keep in mind that !important is not the best choice and shoud be avoided (but it is not always possible, I know)

Reply:
I know, but unfortunately filll does not work with swiper, so that is the only possibility....